tests/FilesystemTrait.php

```diff
@@ -1,17 +1,17 @@
 <?php
 
-namespace OpenFoodFacts;
+namespace OpenFoodFactsTests;
 
 trait FilesystemTrait
 {
-    public function recursiveDeleteDirectory(string $dir): void
+    public static function recursiveDeleteDirectory(string $dir): void
```

Comment:
again version breaking - and as this is a trait -> tests subfolder is the wrong location